lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -656,6 +656,66 @@ describe(getName(), () => {
       });
       expect((await lookup.lookupUpdates(config)).updates).toMatchSnapshot();
     });
+    it('should return pendingChecks', async () => {
+      config.currentValue = '1.4.4';
+      config.depName = 'some/action';
+      config.datasource = datasourceGithubReleases.id;
+      config.stabilityDays = 14;
+      config.internalChecksFilter = 'strict';
+      const yesterday = new Date();
+      yesterday.setDate(yesterday.getDate() - 1);
```

Comment:
Better use luxon DateTime?

Reply:
I think it's simple enough as-is, and just for tests